File: Repaso 6-10-24/1.js
function isCapicua(str) {
  return str.length == 0
    ? true
    : str[0] == str[str.length - 1]
      ? isCapicua(str.slice(1, str.length - 1))
      : false;
}

console.log(isCapicua("oso"));
console.log(isCapicua("reconocer"));
console.log(isCapicua("capicua"));

File: Repaso 6-10-24/1.py
def isCapicua(str):
    if len(str) == 0:
        return True

    return isCapicua(str[1:-1]) if str[0] == str[-1] else False


print(isCapicua("oso"))
print(isCapicua("reconocer"))
print(isCapicua("capicua"))
